dm/tests/all_mode/data/db3.prepare.sql

```diff
@@ -0,0 +1,57 @@
+drop database if exists `all_mode`;
+create database `all_mode`;
+use `all_mode`;
+
+create table t_extract
```

Comment:
please do ALTER TABLE ADD COLUMN with all these JSON function of the two tables in db3.increment.sql, and insert some data